bench/wtperf/wtperf.c

```diff
@@ -970,7 +1134,8 @@ run_mix_schedule(WTPERF *wtperf, WORKLOAD *workp)
 
 	if (workp->truncate != 0) {
 		if (workp->insert != 0 ||
-		    workp->read != 0 || workp->update != 0) {
+		    workp->read != 0 || workp->update != 0 ||
+			workp->modify != 0) {
```

Comment:
Add this alphabetically, and fix the line continuation spacing.

Reply:
corrected.